plugins/sfWebBrowserPlugin/lib/sfCurlAdapter.class.php

```diff
@@ -124,7 +124,15 @@ public function __construct($options = [])
 
     public function __destruct()
     {
-        curl_close($this->curl);
+        $this->close();
+    }
+
+    public function close()
+    {
+        // If curl handle exists, set it to null to close connection
+        if ($this->curl) {
+            $this->curl = null;
```

Comment:
I am afraid that setting a variable to null is not the same as closing... This is a code-smell for a later problem of memory leaks and unused resources.